### DISCUSSION TITLE
How to use execute_script on nodriver ?

### DISCUSSION THREAD

**Discussion by mashien0201** [ultrafunkamsterdam/undetected-chromedriver#1849] [Q&A]
I looked through some nodriver source codes, but I didn’t seem to see any methods related to execute_script. Is it not possible to inject javascript into nodriver?

**Comment by ultrafunkamsterdam** [comment]
Tab.evaluate ,   Tab.js_dumps

**Comment by andvid72** [comment]
@ultrafunkamsterdam please can you give as an example of how use Tab.evaluate or tab.js_dumps to execute JS from nodriver? For example, I need to scroll a div tag section (no the full page).  This code does not work:

div = await tab.select('[class=myclass]')
await Tab.evaluate("arguments[0].scrollBy(0,200);",div)
